shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -56,7 +56,9 @@ void ResetAnchor(CALayer* layer) {
 
 }  // namespace flutter
 
-@implementation ChildClippingView
+@implementation ChildClippingView {
+  NSMutableArray* _gaussianFilters;
```

Comment:
optional nits: use property instead.

Reply:
also maybe name it "activeGaussianFilters"? and give it a comment to explain what it is.